### ISSUE TITLE
Shelf.json troubles: can't access the shelf file

### ISSUE THREAD

**Issue by Bonet83** [Simbul/baker#1010]
Hi!

I´m already updated my Baker Version and it´s great... but when i set the:

`#define NEWSSTAND_MANIFEST_URL @"http://www.mydomain.com/folder/shelf.json"`

The Simulator launches a display message "No Internet Connnection" anything happens and xcode says:

```
ERROR: 'shelf.json' is missing. Add URL to 'NEWSSTAND_MANIFEST_URL' in Constants.h
```

Otherwile when i set the demo directory :

`#define NEWSSTAND_MANIFEST_URL @"http://bakerframework.com/demo/shelf.json"`

It works!!! (But doesn´t loads my own book behind "books" folder).

**Comment by folletto** [comment]
Well, is your `shelf.json` file accessible?

**Comment by fturner19** [comment]
Open up a browser and type in the url where your "shelf.json" file is
located on your server. If you can open the file on your server then
something may have been edited in your baker.xcode file

On Fri, Jul 12, 2013 at 10:32 AM, Davide Casali notifications@github.comwrote:

> Well, is your shelf.json file accessible?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Simbul/baker/issues/1010#issuecomment-20879989
> .

## 

All good things are difficult to achieve; and bad things are very easy to
get.

**Comment by Bonet83** [comment]
Ok, i can´t open my shelf.json file on my browser. What can i do?

**Comment by fturner19** [comment]
First make sure that the file is on your server and then just navigate to
it's url...here's a look at my shelf.json file on my server.
![screen shot 2013-07-12 at 4 33 54 pm](https://f.cloud.github.com/assets/280993/791015/0e3cc12a-eb36-11e2-802d-9e8e0fd3fc4b.png)
![screen shot 2013-07-12 at 4 34 36 pm](https://f.cloud.github.com/assets/280993/791016/0e456d66-eb36-11e2-84fb-81db9113aaab.png)
![screen shot 2013-07-12 at 4 34 52 pm](https://f.cloud.github.com/assets/280993/791017/0e47e6fe-eb36-11e2-9208-01397f1c322d.png)

**Comment by Bonet83** [comment]
Ok, yes the file is in my server but when i type the url on the browser a 404 Error Page Appears.

URL on browser

![captura](https://f.cloud.github.com/assets/4909705/791175/200fb9f2-eb3b-11e2-87e7-2e8bf9a81a05.jpg)

json file on my server
![server](https://f.cloud.github.com/assets/4909705/791177/37c2bec8-eb3b-11e2-861a-c224a81988ce.jpg)

The file is innaccesible...

I think, i have a wordpress installation on my server. Will be the problem?

**Comment by fturner19** [comment]
Contact your hosting company because you should be able to have full
control over the files that you place on your host account.

Just to rule out a few things see if you can access it from a different
hosting account if you have a few of them like myself.

On Fri, Jul 12, 2013 at 5:37 PM, Bonet83 notifications@github.com wrote:

> Ok, yes the file is in my server but when i type the url on the browser a
> 404 Error Page Appears.
> 
> URL on browser
> 
> [image: captura]https://f.cloud.github.com/assets/4909705/791175/200fb9f2-eb3b-11e2-87e7-2e8bf9a81a05.jpg
> 
> json file on my server
> [image: server]https://f.cloud.github.com/assets/4909705/791177/37c2bec8-eb3b-11e2-861a-c224a81988ce.jpg
> 
> The file is innaccesible
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Simbul/baker/issues/1010#issuecomment-20905749
> .

## 

All good things are difficult to achieve; and bad things are very easy to
get.

**Comment by Robotxxx** [comment]
Just a thought, could it be a permissions issue?

**Comment by Bonet83** [comment]
I´ve tried to change the permissions through my FTP but this does not works. I work with Filezilla.

**Comment by Robotxxx** [comment]
Can you view your permissions? It would be worth seeing if that's the problem at least. I've been caught out on that one before!

**Comment by Bonet83** [comment]
Yes, when i try to change permissions trough Fiezilla the answer is: 500 'SITE CHMOD 777 shelf.json': command not understood

**Comment by Robotxxx** [comment]
Have you tried using your host control panel to change or check permissions?

**Comment by Bonet83** [comment]
I contacted my hosting company and I confirm that I have full control over the directory and file, with no security restrictions, but still can not see. Json in the browser.

I do not understand ...

Thanks for all your support.

**Comment by Bonet83** [comment]
Wordpress can be what causes the problem?

**Comment by gegebe** [comment]
I'm with the same problem.

shelf.json is accesible and readable via browser, but when I try to simulate with XCode / iOS Simulator, only appears Baker logo and no issue.

**Comment by folletto** [comment]
Please follow the instructions from the wiki on the page [Problems and Debugging](https://github.com/Simbul/baker/wiki/Problems-and-Debugging) and report back, otherwise it's very difficult to help you. :)

**Comment by Bonet83** [comment]
Hi, there!

I´m Back now. I still have the same propblem with shelf.json... The files isn´t accesible via browser.

**Comment by folletto** [comment]
If your json isn't accessible via browser then it's not a Baker issue, it's a server issue. Contact your service provider or get help from a sysadmin. ;)
